crates/assistant2/src/context_strip.rs

```diff
@@ -44,14 +44,14 @@ impl Render for ContextStrip {
 
         h_flex()
             .flex_wrap()
-            .gap_2()
+            .gap_1()
             .child(
                 PopoverMenu::new("context-picker")
                     .menu(move |_cx| Some(context_picker.clone()))
                     .trigger(
                         IconButton::new("add-context", IconName::Plus)
-                            .shape(IconButtonShape::Square)
```

Comment:
Is there a reason we're removing the `IconButtonShape::Square` here?

We generally use it on icon buttons to ensure they maintain the correct aspect ratio.

Reply:
Check this out:

| No shape, icon small | W/ shape, icon small | W/ shape, icon default |
|--------|--------|--------|
| <img width="233" alt="Screenshot 2024-12-16 at 19 10 23" src="https://github.com/user-attachments/assets/392e2bb0-f2ef-4a39-93c4-db41c69b5b8b" /> | <img width="233" alt="Screenshot 2024-12-16 at 19 11 04" src="https://github.com/user-attachments/assets/a6b8724d-1a8e-48f5-8151-9e860769c8ab" /> | <img width="233" alt="Screenshot 2024-12-16 at 19 11 52" src="https://github.com/user-attachments/assets/5f5c815b-8d15-4269-9eaa-419480580ba0" /> | 

The `shape` property reduces the overall size of the button, and even without it, the button still renders as a square. Then, the "New..." button for both standard panes and the current assistant pane doesn't use `shape`, and I think the balance between icon and container padding is better without it... so figured it'd be safe not to use it!

However I've seen cases where `shape` helped forcing the container to be an actual square. I'm not sure what causes an icon button _not_ to be a square, but I'd suppose it's whenever the SVG itself isn't 1x1?